modules/cli/cli.go

```diff
@@ -22,12 +22,15 @@ func New(m metadata.Info) *CLI {
 
 	var cfg manager.Config
 	var deferCfg flagsForFurtherEvaluation
+	// set default values for MetricsAccessFilter
+	cfg.MetricsAccessFilter = manager.MetricsAccessFilterOff
 
 	flagSet.BoolVar(&cfg.AnonymousReports, "anonymous-reports", true, "Send anonymized usage data to help improve Kong.")
 	flagSet.StringVar(&cfg.APIServerPath, "apiserver-host", "", "The Kubernetes API server URL. If not set, the operator will use cluster config discovery.")
 	flagSet.StringVar(&cfg.KubeconfigPath, "kubeconfig", "", "Path to the kubeconfig file.")
 
 	flagSet.StringVar(&cfg.MetricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flagSet.Var(&cfg.MetricsAccessFilter, "metrics-access-filter", "Specifies the filter access function to be used for accessing the metrics endpoint (possible values: off, rbac).")
```

Comment:
I believe we should mention the default in the description of the flag so that users know what to expect, WDYT?